launch.go

```diff
@@ -204,8 +210,33 @@ func launch(api screwdriver.API, buildID int, rootDir, emitterPath, metaSpace, s
 		return fmt.Errorf("Fetching Pipeline ID %d: %v", j.PipelineID, err)
 	}
 
+	// If no parent build ID, get the parent event meta
 	if b.ParentBuildID == 0 {
-		log.Printf("This build has no Parent Build, so fetching Meta is skipped")
+		log.Printf("Fetching Parent Event %d", e.ParentEventID)
+		pe, err := api.EventFromID(e.ParentEventID)
+		if err != nil {
+			return fmt.Errorf("Fetching Parent Event ID %d: %v", e.ParentEventID, err)
+		}
+
+		// Write to "meta.json" file
+		metaFile := "meta.json"
+
+		log.Printf("Creating Meta Space in %v", metaSpace)
+		err = createMetaSpace(metaSpace)
+		if err != nil {
+			return err
+		}
+
+		log.Printf("Fetching Parent Event Meta JSON %v", pe.ID)
+		peMetaByte, err := marshal(pe.Meta)
+		if err != nil {
+			return fmt.Errorf("Parsing Parent Event(%d) Meta JSON: %v", pe.ID, err)
+		}
+
+		err = writeFile(metaSpace+"/"+metaFile, peMetaByte, 0666)
```

Comment:
`fmt`

Reply:
What is the benefit in using `fmt` vs just doing a simple string concatenation here?